netdata-installer.sh

```diff
@@ -334,7 +334,7 @@ echo >&2
 # run netdata
 
 echo >&2 "Starting netdata..."
-run ${NETDATA_PREFIX}/usr/sbin/netdata "${@}"
+run ${NETDATA_PREFIX}/usr/sbin/netdata -pidfile /var/run/netdata.pid "${@}"
```

Comment:
ok, so we agree that this script should not be used in any supported (packaged) system, so I just ignore it, it has too many issues anyway, such as not using the autoconf vars.